packages/js/e2e-utils-playwright/src/editor.js

```diff
@@ -0,0 +1,164 @@
+const { expect } = require( '@playwright/test' );
+
+export const closeChoosePatternModal = async ( { page } ) => {
+	const closeModal = page
+		.getByLabel( 'Scrollable section' )
+		.filter()
+		.getByRole( 'button', {
+			name: 'Close',
+			exact: true,
+		} );
+	await page.addLocatorHandler( closeModal, async () => {
+		await closeModal.click();
+	} );
+};
+
+export const disableWelcomeModal = async ( { page } ) => {
+	// Further info: https://github.com/woocommerce/woocommerce/pull/45856/
+	await page.waitForLoadState( 'domcontentloaded' );
+
+	const isWelcomeGuideActive = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
```

Comment:
This is relying on a plugin that's installed during wp-env setup, but only in our e2e testing setup.  I worry that putting this into a utility will lead to folks having trouble with this if they're testing against other environments.